dir.traversal.targets

```diff
@@ -77,6 +77,9 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
+  <!-- Dumpling.targets needs to install dumpling.py before running tests. -->
+  <Import Project="$(ToolsDir)Dumpling.targets" Condition="'$(EnableDumpling)' == 'true' OR '$(EnableCloudTest)' == 'true'" />
```

Comment:
could you also add the `Exists('$(ToolsDir)Dumpling.targets')` to your condition? This is only so that if people have not yet ran init-tools they will get a friendly message with instructions of how to run it, as opposed to some error saying that couldn't find the imported project Dumpling.targets